spring-boot/admin/src/main/resources/application.yml

```diff
@@ -83,25 +108,4 @@ spring.cloud.config:
   password: password
   uri: http://localhost:8998
 
-security:
-  oauth2:
-    client:
-      # the following oauth client configuration is just for development testing purposes, do NOT use in a production setting
```

Comment:
Why did you drop this comment? I would consider it important

Reply:
Good point, this seems to be a mistake, it might have happened while removing the old oauth configuration. Based on this, I added again the comment